sparkles/core.py

```diff
@@ -622,15 +623,20 @@ def get_text_pre(self):
         """
         P2 = -np.log10(self.acqs.calc_p_safe())
         att = self.att
+        att_targ = self.att_targ
         self._base_repr_()  # Hack to set default ``format`` for cols as needed
         catalog = '\n'.join(self.pformat(max_width=-1, max_lines=-1))
         self.acq_count = np.sum(self.acqs['p_acq'])
 
+        att_string = f'ACA RA, Dec, Roll (deg): {att.ra:.6f} {att.dec:.5f} {att.roll:.5f}'
```

Comment:
Though I suppose I meant to ask if RA really deserved an extra decimal place compared Dec and Roll.